src/db/tables/onerep_scans.ts

```diff
@@ -33,6 +33,8 @@ export interface LatestOnerepScanDataOld {
   results: OnerepScanResultRow[];
 }
 
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
```

Comment:
Question: It's not directly clear why so many c8 ignore comments were added to existing code. Did new tests import code which imports from these files?

Reply:
Yes, the unit tests in `src/app/functions/server/updateDataBrokerScanProfile.ts` are importing from `src/app/functions/server/refreshStoredScanResults.ts` and `src/db/tables/onerep_scans.ts` which caused the functions to be shown as not covered.